features/unattached_commands.feature

```diff
@@ -218,3 +218,55 @@ Feature: Command behaviour when unattached
           | impish  | disable  |
           | jammy   | enable   |
           | jammy   | disable  |
+
+    @wip
+    @series.all
+    @uses.config.machine_type.lxd.container
+    Scenario Outline: Check for newer versions of the client in an ubuntu machine
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        Then I verify that no files exist matching `/tmp/ua-candidate`
+        When I run `pro status` as non-root
+        Then I will see the following on stderr
+        """
+        """
+        And I verify that files exist matching `/tmp/ua-candidate`
+        When I delete the file `/tmp/ua-candidate`
+        And I create the file `/tmp/ua-candidate` with the following
+        """
+        99.9.9
+        """
+        When I run `pro status` as non-root
+        Then I will see the following on stderr
+        """
+        *Warning*: There is a new version of the client available (99.9.9
+        ).
```

Comment:
What's causing this to go onto another line?

Reply:
yeah took some time for me to see: an unintended \n... Will fix